apps/marketplace/common-util/Details/ActivityDetails.tsx

```diff
@@ -1,12 +1,12 @@
 import type { Activity } from 'common-util/graphql/service-activity';
-import { Flex, Modal, Typography } from 'antd';
+import { Col, Flex, Modal, Row, Typography } from 'antd';
 import { AddressLink, NA } from '@autonolas/frontend-library';
```

Comment:
it would be nice to use the existing AddressLink with some extensions (the ability to copy), the on in this library doesn't have correct scan link for mode. can you possibly fix this as part of this cleanup?